client/driver/qemu_test.go

```diff
@@ -123,44 +126,49 @@ func TestQemuDriver_StartOpen_Wait(t *testing.T) {
 
 func TestQemuDriver_GracefulShutdown(t *testing.T) {
 	logger := testLogger()
+	logger.Printf("TestQemuDriver_GracefulShutdown ...")
+	testutil.SkipSlow(t)
 	if !testutil.IsTravis() {
 		t.Parallel()
 	}
 	ctestutils.QemuCompatible(t)
-	ctestutils.RequireRoot(t)
+	logger.Printf("TestQemuDriver_GracefulShutdown ... RUNNING")
+
+	// Graceful shutdown may be really slow unfortunately
+	killTimeout := 3 * time.Minute
+
+	// Grab a free port so we can tell when the image has started
+	port := freeport.GetT(t, 1)[0]
+
 	task := &structs.Task{
-		Name:   "linux",
+		Name:   "alpine-shutdown-test",
 		Driver: "qemu",
 		Config: map[string]interface{}{
-			"image_path":        "linux-0.2.img",
-			"accelerator":       "tcg",
```

Comment:
what is accelerator and why do we not need it anymore

Reply:
It is the default if accelerator is not specified: https://github.com/hashicorp/nomad/blob/32174fd7298ae66fec4afcdd94042a1207d6d0c4/client/driver/qemu.go#L218-L221

QEMU has a number of accelerators depending on the host and vm architectures. I _think_ tcg is the best you can get when you run in as many different environments as our tests do (linux hosts, virtualbox, vmware, travis).

At any rate this test shouldn't have anything to do with the accelerator being used, so I'd rather let it use the default than imply the test actually cares about the accelerator by specifying it.